### PR TITLE
blog vs website docs

### LINKED ISSUE #138: In the docs website, make it clearer how to choose between website and blog?
I.e. maybe add a paragraph at the beginning of "creating a website" and "creating a blog" explaining what the difference is between the two creation processes in practice, and what difference you make between the two types of websites? I am assuming the difference is on "dating content".

### PR DESCRIPTION
**Pull request by maelle**
Fix #138 

I wasn't able to knit index.Rmd :cry: I got the error below that I can't investigate right now (I see I have the image file) but I could once we agree on the Rmd.

```r
Error in include_graphics("images/l-middle-figure.png") : 
  Cannot find the file(s): "images/l-middle-figure.png"
Calls: <Anonymous> ... withCallingHandlers -> withVisible -> eval -> eval -> include_graphics
Execution halted
```

### CODE REVIEW

File: docs/_common/website_vs_blog.Rmd [hunk 1]
```diff
@@ -0,0 +1,5 @@
+### Website or blog?
+
+Structure difference: [Websites](website.html) are just collections of pages you can navigate to via the menu, whereas [blogs](blog.html) have collections of posts that can be indexed and syndicated (via their RSS feed). Distill blogs are a distill website with added blog posts.
+
+Workflow difference: Furthermore, _website_ pages and root pages of _blogs_ are re-rendered when the site is rebuilt but blog articles are not. Each blog article has to be rendered on its own, with intent. Why? Continuously re-rendering old content is in practice impossible to do for any non-trivial set of articles written over a period of time.
```
Comment (maelle) [inline docs/_common/website_vs_blog.Rmd:5]: Not that it matters, but I wanted to underline that I really like this aspect of distill, and hugodown.

**Comment by jjallaire** [comment]
It looks like there was a change in knitr as to how the working directory in included Rmds is established. I pushed a fix to master so now index.Rmd should render correctly.

File: docs/_common/website_vs_blog.Rmd (hunk 1 above)
Comment (jjallaire) [reply docs/_common/website_vs_blog.Rmd:5]: Maybe just emphasize why here. Perhaps say:

> Why? Given that R package upgrades have a tendency to break older code, continuously re-rendering old posts is nearly impossible to do without errors, especially over longer periods of time.

**Comment by maelle** [comment]
I updated the PR but it seems there are many changed files. :raised_eyebrow:

**Comment by jjallaire** [comment]
Okay, I'll merge this locally then regenerate

**Comment by maelle** [comment]
Sorry! It seems to be due to linebreaks of some other OS specific thing. :grimacing:

**Pull request merged**